### PR TITLE
write project/SmartPointerPerformance to compare share_ptr vs c++ pointer

### PR DESCRIPTION
**Pull request by walzer**

### CODE REVIEW

**Comment by walzer** [comment]
Just show Beijing workmates how to use redmine + github workflow :)

File: projects/SmartPointerPerformance/Classes/AppDelegate.cpp [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+AppDelegate::~AppDelegate() 
+{
+}
+
+bool AppDelegate::applicationDidFinishLaunching() {
+    // initialize director
+    CCDirector* pDirector = CCDirector::sharedDirector();
+    CCEGLView* pEGLView = CCEGLView::sharedOpenGLView();
```
Comment (dumganhar) [inline projects/SmartPointerPerformance/Classes/AppDelegate.cpp:17]: We have deleted `CC` prefix, so this project can't be compiled successfully.

BTW, is it suitable to make this test project to `project` folder ?

**Comment by dumganhar** [comment]
> Just show Beijing workmates how to use redmine + github workflow :)

All right.

File: cocos2dx/Android.mk [hunk 2]
```diff
@@ -168,6 +168,7 @@ LOCAL_WHOLE_STATIC_LIBRARIES += cocos_libwebp_static
 
 # define the macro to compile through support/zip_support/ioapi.c
 LOCAL_CFLAGS := -Wno-psabi -DUSE_FILE32API
+LOCAL_CPPFLAGS += -std=gnu++0x
```
Comment (walzer) [inline cocos2dx/Android.mk:171]: Why you open c++11? Will it break the compatibility with iOS 4.x?

Comment (dumganhar) [inline cocos2dx/Android.mk:171]: Hey, it's your changes.
And it's `Android.mk`, not iOS project file.
iOS supports c++11 from iOS5.0.

**Pull request closed without merging**